modules/nw-creating-a-route.adoc

```diff
@@ -68,12 +68,14 @@ metadata:
 spec:
   host: hello-openshift-hello-openshift.<Ingress_Domain> <1>
   port:
-    targetPort: 8080
+    targetPort: 8080 <2>
   to:
     kind: Service
     name: hello-openshift
 ----
 <1> `<Ingress_Domain>` is the default ingress domain name.
+<2> `targetPort` is the target port on pods selected by the service this route points to.
```

Comment:
This reads a little weird to me, so maybe:

"`targetPort` is the port on pods where this route points. The service selects this port."

Reply:
re-phrased